src/components/CreateNewInvoiceButton/index.jsx

```diff
@@ -0,0 +1,41 @@
+function CreateNewInvoiceButton({invoiceObj, total, clientId}) {
+
+    let formInput = {
+        "client": clientId,
+        "total": total,
+        "details": invoiceObj
+    }
```

Comment:
This is fine, although the props is already an object with these three keys in, but with different names. Could be a bit redundant, but I see why you're doing it.